library/std/src/collections/hash/map.rs

```diff
@@ -136,7 +136,7 @@ use crate::sys;
 /// ]);
 /// ```
 ///
-/// `HashMap` implements an [`Entry API`](#method.entry), which allows
+/// `HashMap` implements an [Entry API](#method.entry), which allows
```

Comment:
```suggestion
/// `HashMap` implements an [`Entry` API](#method.entry), which allows
```

`Entry` is a type, so wrapping it in backquotes makes sense. But I agree that "API" shouldn't be.